solutions/standard/main.tf

```diff
@@ -232,3 +232,81 @@ data "ibm_database_connection" "existing_connection" {
   user_id       = data.ibm_database.existing_db_instance[0].adminuser
   user_type     = "database"
 }
+
+########################################################################################################################
+# Code Engine Kibana Dashboard instance
+########################################################################################################################
+
+data "http" "es_metadata" {
+  url      = "https://${local.es_username}:${local.es_password}@${local.es_host}:${local.es_port}"
+  insecure = true
+}
+
+locals {
+  es_host         = local.use_existing_db_instance ? data.ibm_database_connection.existing_connection[0].https[0].hosts[0].hostname : module.elasticsearch[0].hostname
+  es_port         = local.use_existing_db_instance ? data.ibm_database_connection.existing_connection[0].https[0].hosts[0].port : module.elasticsearch[0].port
+  es_username     = local.use_existing_db_instance ? data.ibm_database.existing_db_instance[0].adminuser : "admin"
+  es_password     = local.admin_pass
+  es_data         = jsondecode(data.http.es_metadata.response_body)
+  es_full_version = local.es_data.version.number
+}
+
+locals {
+  code_engine_project_name = "${var.prefix}-code-engine-kibana-project"
+  code_engine_app_name     = "${var.prefix}-kibana-app"
+}
+
+module "code_engine_kibana" {
+  source            = "terraform-ibm-modules/code-engine/ibm"
+  version           = "2.0.2"
+  resource_group_id = module.resource_group.resource_group_id
+  project_name      = local.code_engine_project_name
+  secrets = {
+    "${var.prefix}-secret" = {
+      format = "generic"
+      data = {
+        "ELASTICSEARCH_PASSWORD" = local.es_password
+      }
+    }
+  }
+  apps = {
+    (local.code_engine_app_name) = {
+      image_reference     = "docker.elastic.co/kibana/kibana:${local.es_full_version}"
+      image_port          = 5601
+      scale_min_instances = 1
+      scale_max_instances = 5
```

Comment:
I think max should be 3 here by default (3 AZs), and also make it configurable. I am not sure we can really justify the max 5 value here, unless there is a specific reason for choosing it.